recipes-kernel/linux/linux-smelt/defconfig

```diff
@@ -458,8 +455,7 @@ CONFIG_AEABI=y
 # CONFIG_ARCH_SPARSEMEM_DEFAULT is not set
 # CONFIG_ARCH_SELECT_MEMORY_MODEL is not set
 CONFIG_HAVE_ARCH_PFN_VALID=y
-CONFIG_HIGHMEM=y
-# CONFIG_HIGHPTE is not set
+# CONFIG_HIGHMEM is not set
```

Comment:
What's the motivation behind disabling CONFIG_HIGHMEM?

Reply:
Mostly assumed though, but here's the description what I found:
```
The address space of ARM processors is only 4 Gigabytes large and it has to accommodate user address space, kernel address space as well as some memory mapped IO. That means that, if you have a large amount of physical memory and/or IO, not all of the memory can be "permanently mapped" by the kernel. The physical memory that is not permanently mapped is called "high memory".

Depending on the selected kernel/user memory split, minimum vmalloc space and actual amount of RAM, you may not need this option which should result in a slightly faster kernel.
```

The Moto 360 only has 512 Megabytes of RAM, so I simply assumed that this option wouldn't be needed. Not entirely sure though.

Reply:
I imagine this is really a nano-optimization but if you didn't notice any weird behaviour I'm ok with it.